docs/how-to-guides/authentication-with-dex-gangway.md

```diff
@@ -250,55 +250,31 @@ Configuring an API server to use the OpenID Connect authentication plugin requir
 
 * Dex is accessible to both your browser and the Kubernetes API server.
 
-To reconfigure the API server with specific flags, edit the `kube-apiserver` DaemonSet as follows:
+To reconfigure the API server with specific flags, add following snippet to your cluster configuration:
 
-```bash
-kubectl -n kube-system edit daemonset kube-apiserver
-```
-
-Add the following CLI arguments to the API server pod:
-
-```bash
---oidc-issuer-url=https://dex.YOUR.CLUSTER.DOMAIN.NAME
---oidc-client-id=gangway
---oidc-username-claim=email
---oidc-groups-claim=groups
+```hcl
+cluster "aws" {
+  ...
+
+  oidc {
+    issuer_url     = https://dex.YOUR.CLUSTER.DOMAIN.NAME
```

Comment:
`s/ https://dex.YOUR.CLUSTER.DOMAIN.NAME / https://dex.<your-clustername>.<your-domain-name> /g`

So as to identify how these are two variables and not four different variables,

Reply:
Ditto in other places.